builders/container-apps-internal-registry-demo/startup-script.sh

```diff
@@ -29,6 +29,20 @@ mkdir -p $CNB_APP_DIR
 cd $CNB_APP_DIR
 tar -xzf "$temp_app_source_path"
 
+fileCount=$(ls | wc -l)
+if [ "$fileCount" = "1" ]; then
+  # Find .jar file in the directory
+  jarfile=$(find "$CNB_APP_DIR" -maxdepth 1 -name "*.jar" | head -n 1)
```

Comment:
do we also need to support `war` and `zip` file extension as well?

Reply:
war will be supported soon, zip is not support.